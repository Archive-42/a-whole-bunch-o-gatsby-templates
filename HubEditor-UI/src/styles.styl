/* You can add global styles to this file, and also import other style files */
html, body{
    height:100%;
    width:100%;
    flex:1;
    display:flex;
}

.f{
    display:flex;
    flex:1;
}

.c{
    flex-direction:column;
}

.r{
    flex-direction:row;
}