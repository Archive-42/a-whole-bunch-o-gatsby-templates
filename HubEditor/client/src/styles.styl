/* You can add global styles to this file, and also import other style files */

@import '~angular-tree-component/dist/angular-tree-component.css';

html, body{
    height:100%;
    width:100%;
    flex:1;
    display:flex;
}

.f{
    display:flex;
    flex:1;
}

.c{
    flex-direction:column;
}

.r{
    flex-direction:row;
}