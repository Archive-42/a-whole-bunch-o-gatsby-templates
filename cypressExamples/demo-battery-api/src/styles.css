/*! normalize.css v3.0.2 | MIT License | git.io/normalize */html{font-family:sans-serif;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block;vertical-align:baseline}audio:not([controls]){display:none;height:0}[hidden],template{display:none}a{background-color:transparent}a:active,a:hover{outline:0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:700}dfn{font-style:italic}h1{font-size:2em;margin:.67em 0}mark{background:#ff0;color:#000}small{font-size:80%}sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}img{border:0}svg:not(:root){overflow:hidden}figure{margin:1em 40px}hr{-moz-box-sizing:content-box;box-sizing:content-box;height:0}pre{overflow:auto}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}button,input,optgroup,select,textarea{color:inherit;font:inherit;margin:0}button{overflow:visible}button,select{text-transform:none}button,html input[type=button],input[type=reset],input[type=submit]{-webkit-appearance:button;cursor:pointer}button[disabled],html input[disabled]{cursor:default}button::-moz-focus-inner,input::-moz-focus-inner{border:0;padding:0}input{line-height:normal}input[type=checkbox],input[type=radio]{box-sizing:border-box;padding:0}input[type=number]::-webkit-inner-spin-button,input[type=number]::-webkit-outer-spin-button{height:auto}input[type=search]{-webkit-appearance:textfield;-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box}input[type=search]::-webkit-search-cancel-button,input[type=search]::-webkit-search-decoration{-webkit-appearance:none}fieldset{border:1px solid silver;margin:0 2px;padding:.35em .625em .75em}legend{border:0;padding:0}textarea{overflow:auto}optgroup{font-weight:700}table{border-collapse:collapse;border-spacing:0}td,th{padding:0}
/**
 * Mobile First
 */
body {
    font: 100%/1.4em "Helvetica Neue", "Helvetica", "Arial", sans-serif;
    margin: 0 auto;
    padding: 0 0.625em;
    color: #444;
    -webkit-text-size-adjust: none;
}

/**
 * Small Screens
 */

.demo-header {
  margin-bottom: 80px;
  text-align: center;
}

.demo-title {
  font-size: 4em;
  line-height: 1em;
  text-align: center;
}

.battery-card {
  font-family: "Helvetica", Arial, sans-serif;
  display: block;
  width: 300px;
  overflow: hidden;
  border: 1px solid #D5D5D5;
  border-radius: 6px;
  font-weight: 100;
  margin: 0 auto;
}

.battery-title {
  background: #4c4c4c url('bolt.png') no-repeat 95% 15px;
  color: #fff;
  font-size: .9em;
  line-height: 50px;
  padding: 0 15px;
  font-weight: 100;
  margin: 0;
}

.battery-percentage {
  font-size: 2.5em;
  line-height: 50px;
  display: inline-block;
  vertical-align: middle;
  margin-right: 10px;
}

.battery-box {
  margin: 0 auto;
  padding: 50px 0;
  text-align: center;
  border-bottom: 1px solid #D5D5D5;
}

.battery {
  display: inline-block;
  position: relative;
  border: 4px solid #000;
  width: 85px;
  height: 40px;
  border-radius: 4px;
  vertical-align: middle;
}

.battery:before {
  content: '';
  display: block;
  box-sizing: border-box;
  background: #000;
  height: 40px;
  position: absolute;
  border: 1px solid #fff;
}

.battery:after {
  content: '';
  display: block;
  background: #000;
  width: 6px;
  height: 16px;
  position: absolute;
  top: 50%;
  right: -11px;
  margin-top: -8px;
  border-radius: 0 4px 4px 0;
}

.battery-info {
  font-size: 12px;
  margin: 0 auto;
  padding: 15px 45px;
  overflow: hidden;
}

.battery-info dd {
  float: right;
  margin-top: -22px;
  text-align: left;
  width: 35%;
}

footer {
  margin: 70px auto 0;
  text-align: center;
}

.heart {
  font-style: normal;
  font-weight: 500;
  color: #c0392b;
  text-decoration: none;
}

#github-button {
  display: block;
  margin: 30px auto 0;
  position: relative;
  left: 40px;
}

#github-ribbon {
  display: inline-block;
  position: fixed;
  top: 0;
  right: 0;
  z-index: 100;
  border: 0;
  width: 149px;
  height: 149px;
}

.github-buttons {
  text-align: center;
  margin: 1em 0;
}



/**
 * Medium Screens
 */
@media all and (min-width:40em) {}

/**
 * Large Screens
 */
@media all and (min-width: 54em) {}