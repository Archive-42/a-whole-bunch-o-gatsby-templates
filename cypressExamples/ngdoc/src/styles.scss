/*eric meyer's css reset for across browser consistency*/
@import "assets/theme/css/reset.css";

/* our colors to be used across the site*/
@import "assets/theme/css/_variables.scss";

@import "assets/theme/css/_buttons.scss";
@import "assets/theme/css/_forms.scss";
@import "assets/theme/css/_tags.scss";

body, html {
  background: $light-grey;
  font-size: 14px;
  font-family: "Roboto", "Trebuchet MS", Arial, Helvetica, sans-serif;
  -webkit-tap-highlight-color: transparent;
  -webkit-touch-callout: none;
  min-height: 100%;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  * {
    box-sizing: border-box;
  }
}
h1 {
  font-size: 2rem;
  line-height: 3rem;
}
a {
  text-decoration: none;
  font-size: 1.2rem;
  line-height: 2rem;
  color: $black;
  cursor: pointer;
  transition: all .4s ease;
  &:hover {
    color: $black;
    text-decoration: underline;
  }
  &:active {
    color: $dark-green;
  }
}

.fa-twitter {
  color: $twitter-blue;
}
