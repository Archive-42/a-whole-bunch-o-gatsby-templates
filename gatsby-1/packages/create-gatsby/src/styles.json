{
  "gatsby-plugin-sass": { "message": "Sass", "dependencies": ["sass"] },
  "gatsby-plugin-styled-components": {
    "message": "styled-components",
    "dependencies": ["styled-components", "babel-plugin-styled-components"]
  },
  "gatsby-plugin-emotion": {
    "message": "Emotion",
    "dependencies": ["@emotion/react", "@emotion/styled"]
  },
  "gatsby-plugin-postcss": {
    "message": "PostCSS",
    "dependencies": ["postcss"]
  },
  "gatsby-plugin-theme-ui": {
    "message": "Theme UI",
    "dependencies": ["theme-ui"]
  }
}
