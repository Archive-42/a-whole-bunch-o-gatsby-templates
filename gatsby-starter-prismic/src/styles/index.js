import prism from './prism'
import theme from './theme'
import reset from './reset'

export { theme, reset, prism }
