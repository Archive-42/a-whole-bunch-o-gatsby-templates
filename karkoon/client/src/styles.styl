/* You can add global styles to this file, and also import other style files */
h2
  color blue // just to prove stylus works
