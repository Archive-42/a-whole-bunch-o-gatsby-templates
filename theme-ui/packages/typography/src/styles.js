// theme.styles object for use with typography.js-generated theme object
// similar to typography.js style output, with these differences
// - only includes styles for markdown elements
// - does not include color styles
// - does not include responsive styles

import assign from 'object-assign'

const heading = {
  fontFamily: 'heading',
  lineHeight: 'heading',
  fontWeight: 'heading',
}

export const styles = {
  root: {
    fontFamily: 'body',
    fontSize: 2,
    fontWeight: 'body',
    lineHeight: 'body',
  },
  img: {
    maxWidth: '100%',
  },
  h1: assign(
    {
      fontSize: 5,
    },
    heading
  ),
  h2: assign(
    {
      fontSize: 4,
    },
    heading
  ),
  h3: assign(
    {
      fontSize: 3,
    },
    heading
  ),
  h4: assign(
    {
      fontSize: 2,
    },
    heading
  ),
  h5: assign(
    {
      fontSize: 1,
    },
    heading
  ),
  h6: assign(
    {
      fontSize: 0,
    },
    heading
  ),
  ul: {
    listStylePosition: 'outside',
    listStyleImage: 'none',
    ml: 3,
  },
  ol: {
    listStylePosition: 'outside',
    listStyleImage: 'none',
    ml: 3,
  },
  li: {
    mb: 2,
    pl: 0,
    ol: {
      my: 2,
      ml: 3,
    },
    ul: {
      my: 2,
      ml: 3,
    },
    p: {
      mb: 2,
    },
  },
  p: {},
  table: {
    borderCollapse: 'collapse',
    width: '100%',
  },
  th: {
    textAlign: 'left',
    borderBottom: '1px solid',
    px: 2,
    py: 1,
    ':first-child': {
      pl: 0,
    },
    ':last-child': {
      pr: 0,
    },
  },
  td: {
    textAlign: 'left',
    borderBottom: '1px solid',
    px: 2,
    py: 1,
    mt: '-1px',
    ':first-child': {
      pl: 0,
    },
    ':last-child': {
      pr: 0,
    },
  },
  blockquote: {
    mx: 3,
  },
  hr: {
    border: 0,
    borderBottom: '1px solid',
    mt: '-1px',
    mb: 3,
  },
  b: {
    fontWeight: 'bold',
  },
  strong: {
    fontWeight: 'bold',
  },
  code: {
    fontSize: '85%',
  },
  pre: {
    fontSize: '85%',
    padding: 3,
  },
}

const headings = ['h6', 'h5', 'h4', 'h3', 'h2', 'h1']
const blockElements = [
  ...headings,
  'ul',
  'ol',
  'p',
  'pre',
  'table',
  'blockquote',
  'img',
  'hr',
]

blockElements.forEach(tag => {
  assign(styles, {
    [tag]: assign(
      {
        padding: 0,
        margin: 0,
        marginBottom: 3,
      },
      styles[tag]
    ),
  })
})

export default styles
