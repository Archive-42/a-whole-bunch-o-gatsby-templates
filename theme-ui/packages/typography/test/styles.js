import styles from '../src/styles'

test('snapshot', () => {
  expect(styles).toMatchSnapshot()
})
