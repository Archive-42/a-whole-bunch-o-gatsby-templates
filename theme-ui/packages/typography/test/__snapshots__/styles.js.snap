// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
Object {
  "b": Object {
    "fontWeight": "bold",
  },
  "blockquote": Object {
    "margin": 0,
    "marginBottom": 3,
    "mx": 3,
    "padding": 0,
  },
  "code": Object {
    "fontSize": "85%",
  },
  "h1": Object {
    "fontFamily": "heading",
    "fontSize": 5,
    "fontWeight": "heading",
    "lineHeight": "heading",
    "margin": 0,
    "marginBottom": 3,
    "padding": 0,
  },
  "h2": Object {
    "fontFamily": "heading",
    "fontSize": 4,
    "fontWeight": "heading",
    "lineHeight": "heading",
    "margin": 0,
    "marginBottom": 3,
    "padding": 0,
  },
  "h3": Object {
    "fontFamily": "heading",
    "fontSize": 3,
    "fontWeight": "heading",
    "lineHeight": "heading",
    "margin": 0,
    "marginBottom": 3,
    "padding": 0,
  },
  "h4": Object {
    "fontFamily": "heading",
    "fontSize": 2,
    "fontWeight": "heading",
    "lineHeight": "heading",
    "margin": 0,
    "marginBottom": 3,
    "padding": 0,
  },
  "h5": Object {
    "fontFamily": "heading",
    "fontSize": 1,
    "fontWeight": "heading",
    "lineHeight": "heading",
    "margin": 0,
    "marginBottom": 3,
    "padding": 0,
  },
  "h6": Object {
    "fontFamily": "heading",
    "fontSize": 0,
    "fontWeight": "heading",
    "lineHeight": "heading",
    "margin": 0,
    "marginBottom": 3,
    "padding": 0,
  },
  "hr": Object {
    "border": 0,
    "borderBottom": "1px solid",
    "margin": 0,
    "marginBottom": 3,
    "mb": 3,
    "mt": "-1px",
    "padding": 0,
  },
  "img": Object {
    "margin": 0,
    "marginBottom": 3,
    "maxWidth": "100%",
    "padding": 0,
  },
  "li": Object {
    "mb": 2,
    "ol": Object {
      "ml": 3,
      "my": 2,
    },
    "p": Object {
      "mb": 2,
    },
    "pl": 0,
    "ul": Object {
      "ml": 3,
      "my": 2,
    },
  },
  "ol": Object {
    "listStyleImage": "none",
    "listStylePosition": "outside",
    "margin": 0,
    "marginBottom": 3,
    "ml": 3,
    "padding": 0,
  },
  "p": Object {
    "margin": 0,
    "marginBottom": 3,
    "padding": 0,
  },
  "pre": Object {
    "fontSize": "85%",
    "margin": 0,
    "marginBottom": 3,
    "padding": 3,
  },
  "root": Object {
    "fontFamily": "body",
    "fontSize": 2,
    "fontWeight": "body",
    "lineHeight": "body",
  },
  "strong": Object {
    "fontWeight": "bold",
  },
  "table": Object {
    "borderCollapse": "collapse",
    "margin": 0,
    "marginBottom": 3,
    "padding": 0,
    "width": "100%",
  },
  "td": Object {
    ":first-child": Object {
      "pl": 0,
    },
    ":last-child": Object {
      "pr": 0,
    },
    "borderBottom": "1px solid",
    "mt": "-1px",
    "px": 2,
    "py": 1,
    "textAlign": "left",
  },
  "th": Object {
    ":first-child": Object {
      "pl": 0,
    },
    ":last-child": Object {
      "pr": 0,
    },
    "borderBottom": "1px solid",
    "px": 2,
    "py": 1,
    "textAlign": "left",
  },
  "ul": Object {
    "listStyleImage": "none",
    "listStylePosition": "outside",
    "margin": 0,
    "marginBottom": 3,
    "ml": 3,
    "padding": 0,
  },
}
`;
